imports/lib/models/public_settings.ts

```diff
@@ -0,0 +1,17 @@
+import { Meteor } from 'meteor/meteor';
+import PublicSettingsSchema, { PublicSettingType } from '../schemas/public_settings';
+import Base from './base';
+
+const PublicSettings = new Base<PublicSettingType>('public_settings');
+PublicSettings.attachSchema(PublicSettingsSchema);
+
+// All public settings are accessible by all clients at all times, including to
+// not-logged-in-users.
+PublicSettings.publish();
```

Comment:
I don't think you can use the `Base#publish` helper for collections that you want to be accessible to non-logged-in-users: https://github.com/deathandmayhem/jolly-roger/blob/e99fd810fafc26b3c0648ec0ac64d6ab7474b4d4/imports/lib/models/base.ts#L139-L141

Probably easiest to just publish it directly

```suggestion
Meteor.publish('mongo.public_settings', () => PublicSettings.find());
```